src/ILCompiler.TypeSystem/tests/VirtualFunctionOverrideTests.cs

```diff
@@ -110,5 +110,25 @@ public void TestFinalizeOverrideChecking()
             Assert.NotNull(actualFinalizer);
             Assert.NotEqual(actualFinalizer, finalizeMethod);
         }
+
+        [Fact]
+        [ActiveIssue(-1)] // Disabled due to https://github.com/dotnet/corert/issues/1302
```

Comment:
Shouldn't we put 1302 instead of -1? I noticed that I did the same for the test I recently added but this was a mistake.